src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -674,6 +684,91 @@ def move_mergeable_units(merge_directory, corpus_directory):
     shell.move(unit_path, dest_path)
 
 
+def pick_strategies(strategy_pool,
+                    fuzzer_path,
+                    corpus_directory,
+                    existing_arguments,
+                    minijail_chroot=None):
+  """Pick strategies."""
+  # Strategy pool is the list of strategies that we attempt to enable, whereas
+  # fuzzing strategies is the list of strategies that are enabled. (e.g. if
+  # mutator is selected in the pool, but not available for a given target, it
+  # would not be added to fuzzing strategies.)
+  build_directory = environment.get_value('BUILD_DIR')
+  target_name = os.path.basename(fuzzer_path)
+  project_qualified_fuzzer_name = data_types.fuzz_target_project_qualified_name(
+      utils.current_project(), target_name)
+
+  fuzzing_strategies = []
+  arguments = []
+  additional_corpus_dirs = []
+
+  # Select a generator to use for existing testcase mutations.
+  generator = _select_generator(strategy_pool, fuzzer_path)
+  is_mutations_run = generator != Generator.NONE
+
+  # Depends on the presense of DFSan instrumented build.
+  dataflow_build_dir = environment.get_value('DATAFLOW_BUILD_DIR')
+  use_dataflow_tracing = (
+      dataflow_build_dir and
+      strategy_pool.do_strategy(strategy.DATAFLOW_TRACING_STRATEGY))
+  if use_dataflow_tracing:
+    dataflow_binary_path = os.path.join(
+        dataflow_build_dir, os.path.relpath(fuzzer_path, build_directory))
+    if os.path.exists(dataflow_binary_path):
+      arguments.append(
+          '%s%s' % (constants.COLLECT_DATA_FLOW_FLAG, dataflow_binary_path))
+      fuzzing_strategies.append(strategy.DATAFLOW_TRACING_STRATEGY.name)
+    else:
+      logs.log_error(
+          'Fuzz target is not found in dataflow build, skiping strategy.')
+      use_dataflow_tracing = False
+
+  use_minijail = environment.get_value('USE_MINIJAIL')
```

Comment:
nit: this can be moved near use - line 735.

Reply:
Remove the variable completely and just used get_value directly in the if.